woodwork/tests/fixtures/datetime_freq.py

```diff
@@ -63,11 +63,10 @@ def pad_datetime_series(dates, freq, pad_start=0, pad_end=100):
             pd.date_range(start=dates[-1], periods=pad_end, freq=freq)[1:]
         ).to_series()
 
-    return (
-        pd.concat([head, pd.Series(dates), tail])
-        .reset_index(drop=True)
-        .astype("datetime64[ns]")
-    )
+    df = pd.concat([head, pd.Series(dates), tail]).reset_index(drop=True)
+    df = pd.to_datetime(df, utc=True)
```

Comment:
Is `utc=True` valid for the non-timezone use cases? I think this is converting all the datetimes to including timezones right? The tests are all passing so I think it's alright but I just wanted to check

Reply:
We ultimately convert everything to UTC anyway right? If we got something in a different timezone would this just convert it to UTC?